cmd/openshift-install/create.go

```diff
@@ -107,6 +113,10 @@ var (
 					logrus.Fatal(err)
 				}
 
+				if err := waitForInitializedCluster(ctx, config); err != nil {
+					logrus.Fatal(err)
+				}
+
 				consoleURL, err := waitForConsole(ctx, config, rootOpts.dir)
```

Comment:
Can we drop this now?  Or can `waitForInitializedCluster` complete before the console route exists?

Reply:
installer being a little resilient when fetching the route doesn't seem too off-the-top. so IMO we can keep the fetch/wait for console route

Reply:
I tried removing the wait--thinking as well that it was no longer necessary. But the console route was still not up at the cluster version became available.